docs/architecture.md

```diff
@@ -6,9 +6,9 @@ Schemas for TIDES suite.
 
 TIDES consists of a package of files as defined in the following table.
 
-**The following table is automatically generated from [`tides.spec.json`](http://github.com/TIDES-transit/blob/main/spec/tides.spec.json)**
+**The following table is automatically generated from [`tides.spec.json`]( https://github.com/TIDES-transit/TIDES/blob/main/spec/tides.spec.json)**
 
-{SPEC}
+{{ frictionless_spec('spec/tides.spec.json') }}
 
 File components for TIDES are specified in [`tides.spec.json`](http://github.com/TIDES-transit/blob/main/spec/tides.spec.json) in a format compatible with the
```

Comment:
and again here!

http://github.com/TIDES-transit/blob/main/spec/tides.spec.json -> https://github.com/TIDES-transit/TIDES/blob/main/spec/tides.spec.json